cmd/signal.go

```diff
@@ -8,8 +8,8 @@ import (
 
 func signalCommand() *cobra.Command {
 	signalCmd := &cobra.Command{
-		Use:   "signal <path> <signal>",
-		Short: "Sends the specified signal to the entry at the specified path",
+		Use:   "signal <signal> [path]...",
```

Comment:
@MikaelSmith If "make supporting multiple paths easy" was your reason for the original `signal <signal> <path` usage, then I apologize for changing it. I didn't think people would want multiple paths so quickly. If you're OK w/ the reasoning in the commit message, then how do you recommend I update the `signal` blog post?

Reply:
I think that is what I was thinking. Exec is still a problem, but I think by it's nature it's going to be hard to make it consistent with everything else.

I'd change the blog post, and add an `Update: ...` note at the top explaining that it's been updated for changes to `signal` in Wash release x.y (once we release).

Reply:
Agreed on exec.

Filed https://github.com/puppetlabs/wash/issues/627 for the blog post update.